### DISCUSSION TITLE
Developer Meeting (May 3 2023): Safety features

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#689] [Announcements]
The next developer meeting will be on the 3rd of May, where we will talk about integrating safety features with brushless platforms with the [Bolt ](https://www.bitcraze.io/products/crazyflie-bolt-1-1/)and [BQ deck](https://www.bitcraze.io/products/bigquad-deck/).

Date: Wednesday May 3 2023 
Time: 3 PM Central European Summer Time  ([check your timezone](https://www.worldtimebuddy.com/?qm=1&lid=5128581,12,5809844,2692969,100,0&h=5128581&date=2023-5-3&sln=9-10&hf=1))
Topic: Safety measures
Hosts: @krichardsson and @Bananazooka 
Recording: https://youtu.be/jQM66zD71gw

Agenda:
15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**This meeting has already passed**

**Comment by knmcguire** [comment]
This blogpost is the base for this meeting https://www.bitcraze.io/2023/04/safety-and-the-brushless/

**Reply by matejkarasek** [reply]
I am in! 

PS: the meeting link above is broken (but copying and pasting the text works...)

**Reply by knmcguire** [reply]
ah thanks! 

Weird... I've changed the url now to include https:// but didn't think that would be really necessary

**Comment by knmcguire** [comment]
Recording can be found here: Recording: https://youtu.be/jQM66zD71gw
